classes/services/queryBuilders/PKPSubmissionListQueryBuilder.inc.php

```diff
@@ -43,6 +43,9 @@
 	/** @var string search phrase */
 	protected $searchPhrase = null;
 
+	/** @var string return a Submission or PublishedArticle\PublishedMonograph */
```

Comment:
Should be `@var string|null`, to be reeeeeally pedantic

Reply:
Pedantic is what we're paid for. :)